### ISSUE TITLE
can't mount vboxfs because vbguest plugin fails to install vboxadditions

### ISSUE THREAD

**Issue by itsuugo** [CentOS/sig-cloud-instance-build#89]
The vbguest plugin fails to install the vboxadditions because can't install the package for the kernel headers 
`No package kernel-devel-3.10.0-327.el7.x86_64 available.`

It started to fail 2 days ago, I guess that we need a base box with the new kernel version 3.10.0-514.2.2.el7

```[root@localhost ~]# uname -a
Linux localhost.localdomain 3.10.0-327.36.3.el7.x86_64 #1 SMP Mon Oct 24 16:09:20 UTC 2016 x86_64 x86_64 x86_64 GNU/Linux
[root@localhost ~]# yum install kernel-devel
Loaded plugins: fastestmirror
Loading mirror speeds from cached hostfile
 * base: www.ftp.ne.jp
 * extras: www.ftp.ne.jp
 * updates: www.ftp.ne.jp
Resolving Dependencies
--> Running transaction check
---> Package kernel-devel.x86_64 0:3.10.0-514.2.2.el7 will be installed
--> Finished Dependency Resolution```

**Comment by lpancescu** [comment]
We now have Vagrant release v1611 available on Atlas, which is based on 7.3.1611 and will help you avoid this problem for now.

This is not really an issue with the Vagrant images, but part of how point releases work. `vagrant-vbguest` could avoid this issue by either installing the corresponding `kernel-devel` package from CentOS Vault if the box it is trying to create was based on an older point release, or upgrade and reboot the box *before* trying to install the VirtualBox Guest Additions. My [starter kit][1] uses the latter approach and worked without any problems with the existing v1610 box, both before and after 7.3.1611 was released.

[1]: https://github.com/lpancescu/cloud-instance-starter-kit

I would propose closing this bug, since I see it as an issue with how `vagrant-vbguest` works, not with our official Vagrant images.

**Comment by itsuugo** [comment]
Thanks for the explanation,  didn't know about it. It's clear  now that's a problem in the plugin.
